__tests__/app.test.js

```diff
@@ -28,4 +29,44 @@ describe("GET /api/topics", () => {
       });
     });
 
+    describe("GET /api/articles/:article_id", () => {
+        test("should respond with an object with the properties: author, title, article_id, body,topic,created_at & votes.", () => {
```

Comment:
You could put the expected http response code explicitly in the test description to make your test results more verbose.